packages/web-frontend/src/sagas.js

```diff
@@ -745,28 +739,8 @@ function* watchMeasureChange() {
   yield takeLatest(SET_MEASURE, fetchMeasureInfoForMeasureChange);
 }
 
-function* fetchMeasureInfoForMeasurePeriodChange() {
-  const state = yield select();
-  const currentMeasureId = selectCurrentMeasureId(state);
-
-  yield fetchMeasureInfo(currentMeasureId);
-}
-
-function* watchMeasurePeriodChange() {
-  yield takeLatest(UPDATE_MEASURE_CONFIG, fetchMeasureInfoForMeasurePeriodChange);
-}
```

Comment:
Should have mentioned earlier, but this section is not part of the refactor (should be with later commits)